network-api/networkapi/minisites/README.md

```diff
@@ -0,0 +1,142 @@
+# Minisites in Wagtail
+
+This document explains how the wagtail `minisites` app works with respect to the "networkapi" Django/Mezzanine instance.
+
+The base app has the following directory structure:
+```
+- minisites
```

Comment:
Maybe call this `wagtail` instead?

Reply:
I'd rather keep the name related to what the app adds "into" django/wagtail, which right now is all the code necessary for minisites - open to better names based on the content it allows for, but "wagtail" is already the name of the CMS framework so I'd rather not call it that.

Reply:
When we eventually move the entire site into the wagtail tree, will that be done in this app as well? If so, perhaps we should call this something like `wagtailPages`?

Reply:
Ideally we find a name that we don't then change a second time once mezzanine is out of the picture.

Reply:
I'm inclined to agree, but I think "wagtail" here is more of a useful adjective, as "pages" isn't the most useful app name.

Reply:
true, but down the line we're removing that "wagtail" because it's all wagtail, and then we're left with "pages", which is a universally bad name. I'm perfectly happy punting on that and just using `wagtail-pages` but eventually we'll need a better name